astro/src/content/docs/release-notes/index.mdx

```diff
@@ -13,17 +13,119 @@ import ReleaseNoteHeading from 'src/components/docs/release-notes/ReleaseNoteHea
+
+### Fixed
+* Clicking the toggle checkbox element in the admin UI quickly may cause the checkbox state to be inverted. This can be easily fixed by refreshing the page. You should now be able to click as fast as you want! Go forth and click!
+  * Resolves [GitHub Issue #2718](https://github.com/FusionAuth/fusionauth-issues/issues/2718)
+
+* Attempting to sort API keys by key value in the admin UI by clicking the key value header would result in an error.
+  * Resolves [GitHub Issue #2738](https://github.com/FusionAuth/fusionauth-issues/issues/2738)
+
+* When using the API Key API and specifying an invalid `tenantId` on the request in order to create a tenant-scoped API key, the request fails with a `500` status code. This error has been corrected, and an appropriate validation error is now returned.
+  * Resolves [GitHub Issue #2749](https://github.com/FusionAuth/fusionauth-issues/issues/2749)
+
+* The date picker component used in simple themes for things like birthdates was not styled correctly based upon the selected theme. The date picker has been changed to use the browser-default date picker which should work much better on mobile devices. This picker style will now be used in themed hosted login pages, as well as the admin UI for searching a date range or selecting a birthdate. This change should not affect any existing advanced theme that may still be using the older style date picker.
+  * Resolves [GitHub Issue #2770](https://github.com/FusionAuth/fusionauth-issues/issues/2770)
+
+* Adding custom message keys to your theme messages using the admin UI was failing to persist these changed messages. The UI for editing messages in the simple theme editor has also been improved to make it easier to understand which messages have been modified.
+  * Resolves [GitHub Issue #2778](https://github.com/FusionAuth/fusionauth-issues/issues/2778)
+
+* The preview window for simple themes in the admin UI was showing un-styled pages when using the Firefox browser. Sorry Firefox users, we ask for your forgiveness. 😔
+  * Resolves [GitHub Issue #2794](https://github.com/FusionAuth/fusionauth-issues/issues/2794)
+
+* The default `orderBy` parameter value for the [Group Member Search API](/docs/apis/groups#search-for-group-members) did not provide a consistent ordering of results because the default sort was on `insertInstant ASC` which may not always be unique. This API is used by the SCIM Groups Resource API which then can cause inconsistent results for the SCIM client. The default `orderBy` is now set to `insertInstant ASC, userId ASC, groupId ASC` to ensure a consistent result between API calls.
+  * Resolves [GitHub Issue #2798](https://github.com/FusionAuth/fusionauth-issues/issues/2798)
+
+* When using the simple theme editor in the admin UI, the color picker did not always render next to the input field. The color picker will now always correctly render adjacent to the input field you select.
```

Comment:
```suggestion
* When using the simple theme editor in the admin UI the color picker did not always render next to the input field. The color picker should now correctly render adjacent to the appropriate input field you select.
```

`always` is technically untrue and out of our control as the browser gets to decide.